### ISSUE TITLE
Meta Issue: 4.5

### ISSUE THREAD

**Issue by frangio** [OpenZeppelin/openzeppelin-contracts#2900]
**Candidates:**

- #2686 
- #2917
- #2815
- #3039
- #2884
- #3021
- ~https://github.com/OpenZeppelin/openzeppelin-contracts/issues/2901~
  - Blocked by change in type (bool → uint8) that is not supported by the upgrades plugin
  - Postponed to 4.6
- #3035
- #3012
- #3036
- https://github.com/OpenZeppelin/openzeppelin-contracts/issues/3031
- **Governor**
  - #2873
  - #2926
- #3078
- #3085
- #3100

**Comment by Amxx** [comment]
Last addition proposal:
- #3078
- #3085

**Comment by Amxx** [comment]
Security fix #3100

**Comment by Amxx** [comment]
To cherry-pick (fixed during RC)
- #3110
- #3114
- #3133

**Comment by frangio** [comment]
4.5 is now released!
